aws_lambda_powertools/utilities/data_classes/bedrock_agent_event.py

```diff
@@ -97,3 +98,11 @@ def session_attributes(self) -> Dict[str, str]:
     @property
     def prompt_session_attributes(self) -> Dict[str, str]:
         return self["promptSessionAttributes"]
+
+    # For compatibility with BaseProxyEvent
+    @property
+    def path(self) -> str:
+        return self["apiPath"]
+
+    def header_serializer(self) -> BaseHeadersSerializer:
+        return NoopSerializer()
```

Comment:
double check, maybe you don't need it since you're overriding `build`.

Reply:
You're right, removed!